src/DomBuilder/BaseDomBuilder.php

```diff
@@ -72,6 +74,15 @@ public function __construct(string $painFormat, bool $withSchemaLocation = true)
         $this->doc->appendChild($this->root);
     }
 
+    private function setXmlns(string $painFormat)
+    {
+        if (filter_var($painFormat, FILTER_VALIDATE_URL)) {
```

Comment:
I assume you've got a URL from credit suisse, do you mind adding this to the documentation too, so others know how to use this feature ?

Reply:
no problem.

i have to also make a change to 
`$this->root->setAttribute('xsi:schemaLocation', "urn:iso:std:iso:20022:tech:xsd:$painFormat $painFormat.xsd");

to parse the url qad extract the xsd name`